lib/roo_on_rails/railties/database.rb

```diff
@@ -8,6 +8,7 @@ class Database < Rails::Railtie
           config = ActiveRecord::Base.configurations[Rails.env]
           config['variables'] ||= {}
           config['variables']['statement_timeout'] = ENV.fetch('DATABASE_STATEMENT_TIMEOUT', 200)
+          config['reaping_frequency'] = ENV['DB_REAPING_FREQUENCY']
```

Comment:
Maybe we should make this `DATABASE_REAPING_FREQUENCY` for consistency with the `DATABASE_STATEMENT_TIMEOUT` variable? Not a huge deal.